practice-one/src/index.html

```diff
@@ -166,6 +166,62 @@ <h3 class="heading feature-heading">Cashless payment</h3>
         </section>
       </section>
       <!-- END SECTION INTRODUCING -->
+
+      <!-- START SECTION BENEFITS -->
+      <section class="section-benefits">
+        <h2 class="heading benefits-heading">
+          <span class="contrast-heading">Tere</span> benefits
+        </h2>
+        <section class="benefit-card">
+          <hgroup class="card-info">
+            <h3 class="benefit-heading">
+              <span class="heading-number">01.</span>
+              Flexible working hours
+            </h3>
+            <p class="content benefit-content">
+              You can decide when, and how much time you want to drive.
+            </p>
+          </hgroup>
+
+          <figure class="working-hour-img">
```

Comment:
You can use background image for the benefit-card instead of an image here. Let use background position to style position for image